rts/Lua/LuaHandleSynced.cpp

```diff
@@ -2120,16 +2120,187 @@ int CSyncedLuaHandle::GetWatchWeaponDef(lua_State* L) {
 	return 1;
 }
 
+/*** Register/deregister callins working per defID.
+ *
+ * Some of the engine callins can result in so many callins the engine doesn't forward them until registered
+ * through the following SetWatch* methods.
+ *
+ * The GetWatch* methods can be used to query the currently registered defIDs.
+ *
+ * @section watch_methods
+ */
+
+/*** Query watch status for a unitDefID.
+ *
+ * @function Spring.GetWatchUnit
+ *
+ * @param unitDefID integer
+ * @return boolean watched Watch status.
+ *
+ * @see Spring.SetWatchUnit
+ */
+
 GetWatchDef(Unit)
+
+
+/*** Query watch status for a featureDefID.
+ *
+ * @function Spring.GetWatchFeature
+ *
+ * @param featureDefID integer
+ * @return boolean watched Watch status.
```

Comment:
```suggestion
 * @return boolean watched `true` if callins are registered, otherwise `false`.
```

Something like this imo.